solutions/03-run_llm_cpu_vs_gpu.ipynb

```diff
@@ -4,16 +4,49 @@
       "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    # from huggingface_hub import login; login(token="")
</pre><p>I would also leave this out and if needed move to the general setup sction</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PrunaAI/ai-efficiency-courses/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>